src/main/java/evolvio/render/WindowStarter.java

```diff
@@ -0,0 +1,20 @@
+package evolvio.render;
```

Comment:
What about evolviol.ui.view package?
We may need evolviol.ui.listener in future.
Entities rendering can be moved to evolviol.ui.domain or something like this.